backend/routes/texts/index.js

```diff
@@ -28,4 +28,18 @@ router.get('/texts/:textId', async (req, res) => {
   }
 });
 
+router.post('/texts/upload', async (req, res) => {
```

Comment:
POST запрос уже означает загрузку.
```suggestion
router.post('/texts', async (req, res) => {
```